eslint-warning-thresholds.json

```diff
@@ -2,24 +2,24 @@
   "@typescript-eslint/consistent-type-exports": 19,
   "@typescript-eslint/no-base-to-string": 3,
   "@typescript-eslint/no-duplicate-enum-values": 2,
-  "@typescript-eslint/no-unsafe-enum-comparison": 34,
+  "@typescript-eslint/no-unsafe-enum-comparison": 29,
```

Comment:
I don't think any new instances of this rule were introduced, in fact, it seems like there are 2 less now for some reason. So this just needs to be adjusted to 32:

``` suggestion
  "@typescript-eslint/no-unsafe-enum-comparison": 32,
```